ci-operator/step-registry/baremetal/lab/ipi/conf/virtual-media/baremetal-lab-ipi-conf-virtual-media-commands.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -o errtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+if [ "${REDFISH_VIRTUAL_MEDIA:-false}" != "true" ]; then
```

Comment:
If we run this step only in the vmedia workflows, we don't need this variable anymore. So we need an additionl chain, ipi-conf-vmedia, that is the same as the modified version of ipi-conf-chain that you have now, and then restore the previous version of ipi-conf-chain and delete any instance of REDFISH_VIRTUAL_MEDIA